### PR TITLE
Euro calliope native regions and mapping

### PR DESCRIPTION
**Pull request by FLomb**
Following the indications provided in "ECEMF T1.1 Model Diagnostic Exercise – Protocol v2.0", I have added Euro-Calliope 2.0's native regions and their mapping to larger aggregate groups of regions (e.g. EU27). The names I am using for regions in the YAML files in this pull request are the same I am using for the upload in the ECEMF's Scenario Explorer.

### CODE REVIEW

File: definitions/region/model_native_regions/euro-calliope_2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+- Euro-Calliope 2.0:
```
Comment (danielhuppmann) [inline definitions/region/model_native_regions/euro-calliope_2.0.yaml:1]: Please add a comment with a reference to the model documentation, in particular the region list.
See this example: https://github.com/openENTRANCE/openentrance/blob/main/definitions/region/model_native_regions/remind_2.1.yaml

**Comment by danielhuppmann** [comment]
Please take a look at my inline review comment to add some reference to the region documentation of the model.

Also, nomenclature-validation is currently failing because "EU27 (excl. Malta)" and "EU27+UK (excl.Malta)" are not defined regions for this project (see definitions/regions/european-regions.yaml for available aggregate-regions).

**Comment by FLomb** [comment]
Thanks @danielhuppmann, all is clear. I'll be back to the PR asap

**Review by danielhuppmann — APPROVED** [review]
Thanks @FLomb

**Pull request merged**